cardano-api/src/Cardano/Api/Convenience/Construction.hs

```diff
@@ -0,0 +1,102 @@
+-- | Convenience transaction construction functions
+--
+module Cardano.Api.Convenience.Construction (
+    constructBalancedTx,
+
+    -- * Misc
+    TxInsExistError(..),
+    NotScriptLockedTxInsError(..),
+    notScriptLockedTxIns,
+    renderNotScriptLockedTxInsError,
+    renderTxInsExistError,
+    txinsExist,
```

Comment:
`txInsExist` ?

Reply:
Open to naming suggestions: `txInsExistInUTxO`? Or perhaps I shouldn't export it in the first place